hibernate-reactive-quickstart/src/main/resources/application.properties

```diff
@@ -1,10 +1,8 @@
-%prod.quarkus.datasource.db-kind=postgresql
-%prod.quarkus.datasource.username=quarkus_test
-%prod.quarkus.datasource.password=quarkus_test
-
-quarkus.hibernate-orm.database.generation=drop-and-create
 quarkus.hibernate-orm.log.sql=true
 quarkus.hibernate-orm.sql-load-script=import.sql
 
 # Reactive config
 %prod.quarkus.datasource.reactive.url=vertx-reactive:postgresql://localhost/quarkus_test
+%prod.quarkus.datasource.db-kind=postgresql
```

Comment:
For the default datasource, this is not necessary, it's inferred from the classpath.

(For named datasources, it's only necessary because we need _some_ build-time property to be set to give Quarkus a hint that named datasource is configured)

Reply:
https://github.com/quarkusio/quarkus-quickstarts/pull/1520/commits/cbe16ea571f568582931e12c17e9924d49d111e7